### PR TITLE
Override moment relative timestamp translations - Sinhala, Thai, Ukrainian

### LINKED ISSUE #2154: Override moment relative timestamp translations - Sinhala, Thai, Ukrainian
**Is your feature request related to a problem? Please describe.**
Apply the correct wording for relative timestamp to match what we have currently on PAL.

**Describe the solution you'd like**
Override moment's default relative time in `psammead-locales` to have the correct translations.
**Services**
- Sinhala
- Thai
- Ukrainian

**Example**
```js
moment.updateLocale('yo', {
  relativeTime: {
    past: '%s sẹ́yìn',
    s: 'ìṣẹ́jú aayá die',
    m: 'ìṣẹ́jú kan',
    mm: 'ìṣẹ́jú %d',
    h: 'wákàtí kan',
    hh: 'wákàtí %d',
  },
});
```
More details on the spreadsheet. 
**Describe alternatives you've considered**

**Testing notes**
[Tester to complete]

Dev insight: Will Cypress tests be required or are unit tests sufficient? Will there be any potential regression? etc

- [x] This feature is expected to need manual testing.

**Additional context**

### PR DESCRIPTION
**Pull request by OlgaLyubin**
Resolves #2154 

**Overall change:** 
Override moment relative timestamps in `psammead-locales` to have the correct translations for Sinhala, Thai and Ukrainian services.

**Code changes:**
Added Thai and Sinhala locales
Updated Ukrainian relative timestamp

---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] Automated jest tests added (for new features) or updated (for existing features)
- [x] This PR requires manual testing

### CODE REVIEW

**Review by dr3 — APPROVED** [review]

File: packages/utilities/psammead-locales/moment/si.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+var moment = require('moment');
+require('moment/locale/si');
+
+moment.updateLocale('si', {
+  postformat: null,
```
Comment (ghost) [inline packages/utilities/psammead-locales/moment/si.js]: Do we need this?

**Review by ghost — CHANGES_REQUESTED** [review]
This is also supposed to handle the overriding of months for Ukrainian.
`января_февраля_марта_апреля_мая_июня_июля_августа_сентября_октября_ноября_декабря`

**Comment by OlgaLyubin** [comment]
> This is also supposed to handle the overriding of months for Ukrainian.
> `января_февраля_марта_апреля_мая_июня_июля_августа_сентября_октября_ноября_декабря`

We do not need to override months for Ukrainian with these for now because those are Russian and are required for a Russian locale.

File: packages/utilities/psammead-locales/moment/index.stories.jsx [hunk 2]
```diff
@@ -27,11 +27,15 @@ import './pcm';
...
 import './ru';
+import './si';
+import './sr';
 import './rw';
 import './so';
 import './sr-cyrl';
 import './sr';
 import './ta';
+import './th';
+import './gu';
```
Comment (DenisHdz) [inline packages/utilities/psammead-locales/moment/index.stories.jsx]: Remove this line. `gu` is already imported above.

File: packages/utilities/psammead-locales/moment/index.stories.jsx [hunk 3]
```diff
@@ -60,8 +64,11 @@ const locales = [
   { name: 'Russian', locale: 'ru' },
   { name: 'Serbian', locale: 'sr' },
   { name: 'Serbian Cyrillic', locale: 'sr-cyrl' },
-  { name: 'Somali', locale: 'so' },
+  { name: 'Sinhala', locale: 'si' },
   { name: 'Tamil', locale: 'ta' },
+  { name: 'Thai', locale: 'th' },
+  { name: 'Gujarati', locale: 'gu' },
```
Comment (DenisHdz) [inline packages/utilities/psammead-locales/moment/index.stories.jsx]: Same here with `Gujarati`

**Review by DenisHdz — CHANGES_REQUESTED** [review]

File: packages/utilities/psammead-locales/moment/index.stories.jsx [hunk 4]
```diff
@@ -27,11 +27,14 @@ import './pcm';
 import './ps';
 import './pt-br';
 import './ru';
+import './si';
+import './sr';
```
Comment (DenisHdz) [inline packages/utilities/psammead-locales/moment/index.stories.jsx]: Serbian `sr` is also imported twice. Can you check they are all fine and sorted alphabetically, please?

**Review by DenisHdz — CHANGES_REQUESTED** [review]

File: packages/utilities/psammead-locales/moment/index.stories.jsx (hunk 4 above)
Comment (OlgaLyubin) [reply packages/utilities/psammead-locales/moment/index.stories.jsx]: All sorted now, thanks!

**Review by DenisHdz — APPROVED** [review]
Looks good :)

**Review by ghost — APPROVED** [review]

**Comment by paruchurisilpa** [comment]
LGTM

**Pull request merged**